frontend/src/app/components/modals/confirm-dialog/confirm-dialog.modal.sass

```diff
@@ -1,8 +1,5 @@
 
-.op-modal--modal-close-button
-  right: 1.75rem
-  top: 1.6rem
 
-.op-modal--modal-container
-  padding-bottom: 0
-   
\ No newline at end of file
+.confirm-dialog--modal
+  p
+    margin-right: 15px
```

Comment:
I think, it is the wrong way to add a margin to **every** paragraph with the modal because of two reasons: First, the `strong` is not included, so the space is missing there, too. Second, what you actually want to achieve, is that the container has a padding to **all** its content. So it might make sense to either extend [this rule ](https://github.com/opf/openproject/blob/34788ba1b5a85b4f7cfd1c523e4209e592e9c814/app/assets/stylesheets/content/_forms.sass#L96) or even better, introduce a new class so that we get rid of the `*` selector there. We can talk about that one in more detail, if you want to.